js/Wizard.js

```diff
@@ -110,7 +110,7 @@ function usefulName(path) {
 
 function checkSeed(seed) {
     console.log("Checking seed")
-    var wordsArray = lineBreaksToSpaces(seed).split(" ");
+    var wordsArray = lineBreaksToSpaces(seed).split(/\s+/);
```

Comment:
Consider to drop `lineBreaksToSpaces` entirely

Reply:
You are right of course, regular expression `s` alone already nicely covers any kind of whitespace, not only blanks. PR is updated.